### PR TITLE
Demo ready

### PR DESCRIPTION
**Pull request by tsingh777**

### CODE REVIEW

File: ui/app.js [hunk 1]
```diff
@@ -15,48 +26,44 @@ const corsOptions = {
...
-    // /** A better way to copy the uploaded file. **/
-    // const src = fs.createReadStream(tmp_path);
-    // const dest = fs.createWriteStream(target_path);
-    // src.pipe(dest);
-    // src.on('end', function () { res.render('complete'); });
-    // src.on('error', function (err) { res.render('error'); });
+app.post('/classify', type, async (req, res) => {
+    const file = req.file;
 
+    console.log("RECIEVED AUDIO: ", file);
```
Comment (lbk003) [inline ui/app.js]: Nit. Typo: RECEIVED

File: ui/app.js [hunk 2]
```diff
@@ -15,48 +26,44 @@ const corsOptions = {
...
-    //     console.log(fromRunpy.toString());
-    //     res.end(fromRunpy);
-    // });
+    const waveFile = `${file.destination}${Date.now() + '.wav'}`;
+    convertFile(file.path, waveFile, () => {
+        console.log("created: ", waveFile);
+        getInference(waveFile, (data) => {
+            console.log(data);
+            const match = data.split(", ")[0].split("'")[1];
+            const confidence = Number.parseFloat(data.split(", ")[1].split(")")[0])*100;
```
Comment (lbk003) [inline ui/app.js]: The screen recording showed too many decimal points. Can we trim to two?

File: ui/input-app/src/components/microphone.styled.ts [hunk 3]
```diff
@@ -26,6 +26,16 @@ ${buttonCommon}
...
 
-export const MicrophoneContainer = styled.div`
+export const MicrophoneContainer = styled.div<{ recording?: boolean }>`
   border: solid 2px #000;
+  border-radius: 100px;
+  width: 100px;
+  height: 100px;
+
+  canvas {
+    ${({ recording }) => recording ? 'visibility: visiable;' : 'visibility: hidden;'}
```
Comment (lbk003) [inline ui/input-app/src/components/microphone.styled.ts:36]: visibility:visible 

How does it work

**Review by lbk003 — DISMISSED** [review]
Are you going to add logic when the confidence level is low? 

Post hackathon -
Do you know if the file transformation is taking time or the inference? 

Also we are loading numpy in Python prices. There could be an optimization there to make it more lightweight

File: ui/input-app/src/components/microphone.styled.ts (hunk 3 above)
Comment (tsingh777) [reply ui/input-app/src/components/microphone.styled.ts:36]: css

File: ui/app.js (hunk 1 above)
Comment (tsingh777) [reply ui/app.js]: just a log.

**Pull request merged**